internal/v2/s2av2.go

```diff
@@ -69,14 +71,24 @@ func (c *s2av2TransportCreds) ClientHandshake(ctx context.Context, serverAuthori
 	if err != nil {
 		return nil, nil, err
 	}
+	// Create an AccessTokenManager instance to use to authenticate to S2Av2.
+	os.Setenv("S2A_ACCESS_TOKEN", "token")
```

Comment:
We don't want to set this environment variable in the actual source code - it may or may not be set by the Cloud infrastructure depending on where the application is running.